### ISSUE TITLE
Option to filter results

### ISSUE THREAD

**Issue by pklaschka** [server-state/linux-raid-module#5]
It might be useful to add an option to filter the partitions whose data gets returned.

#### (Possible?) Use Case
In company X, there are two groups of users of their server-state instance. Designers, who use the RAID for project files, and administrators, who have full control over the servers.

To simplify usage for designers, the linux-raid-module allows filtering the data returned server-side. With a simple configuration like

```js
server.addModule('raid-designer', raid, ['partition 1'])
server.addModule('raid-admin', raid);
```

designers only get shown `partition 1`, which is the one they care about, while admins still get all the information they need. Furthermore, this enhances security as information about the server architecture can remain on a need-to-know basis (for whatever that's worth :wink: )

**Comment by fussel178** [comment]
Added in pull request #9

**Comment by fussel178** [comment]
Implemented with current pull request #9.
